cmd/bancheck/config/config_test.go

```diff
@@ -0,0 +1,214 @@
+package config
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"golang.org/x/tools/go/analysis/analysistest"
+)
+
+func TestBannedFunctionAnalyzer(t *testing.T) {
+	tests := []struct {
+		desc   string            // describes the test case
+		files  map[string]string // fake workspace files
+		config *Config           // the expected config
```

Comment:
rename to `want` and drop all the comments, should be self explanatory then

Reply:
This has been replaced with the built in test method for the analysis package.